code/game/objects/items/storage/belt.dm

```diff
@@ -299,7 +299,7 @@
 		if(M.current_rounds)
 			if(contents.len < storage_slots)
 				to_chat(user, "<span class='notice'>You start refilling [src] with [M].</span>")
-				if(!do_after(user, 15, TRUE, 5, BUSY_ICON_GENERIC))
```

Comment:
Is the removal of the icon intended here?

Reply:
Not a visible message and fairly short delay.